lib/testing/src/lib/core/pages/data-table-component.page.ts

```diff
@@ -274,6 +282,10 @@ export class DataTableComponentPage {
         return this;
     }
 
+    waitTillContentLoaded() {
```

Comment:
is this method being used anywhere?

Reply:
is a public method I can not be sure about it